src/Eshopworld.Messaging/Messenger.cs

```diff
@@ -106,17 +118,30 @@ public async Task Subscribe<T>(Action<T> callback, string subscriptionName, int
         }
 
         /// <inheritdoc />
-        public void CancelReceive<T>()
-            where T : class
+        public async Task Subscribe<T>(Action<T> callback, string subscriptionName, string topicName, int batchSize = 10) where T : class

```

Comment:
Should we share the code for both `Subscribe` methods? Existing one is just very specific usage of the new, more generic one - no reason to duplicate the code?

Reply:
yes, will do